torch/onnx/symbolic_opset9.py

```diff
@@ -4298,7 +4298,9 @@ def linalg_vector_norm(g, self, ord, dim, keepdim, dtype):
     # Conditions based on https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html
     if dim is None:
         self = symbolic_helper._reshape_helper(g, self, [-1])
-        keepdim = None
+        keepdim = 0
+    else:
+        keepdim = int(keepdim)
```

Comment:
Q: Why is this needed?

Reply:
In case of `keepdim` is `None`, we cast it to `0`. ONNX seems to treat `None` and `0` as different.